### DISCUSSION TITLE
Jersey 404 request logs on edge pods

### DISCUSSION THREAD

**Discussion by reubenhEROAD** [featurehub-io/featurehub#987] [Q&A] [answered]
I have deployed featurehub 1.6.0 using helm to EKS. 

The edge pods are consistently logging these 404 requests, around 50 per minute across the two pods. It is not caused by attempted client connections, I have confirmed this by completely redeploying with no clients attempting to connect yet.

```
Failed jersey request
jakarta.ws.rs.NotFoundException: HTTP 404 Not Found
```

The log and stack trace don't really have any more useful information on where the request is coming from or what is being requested. Can someone try help me figure out what is causing this?

**Comment by chrusty** [comment]
@reubenhEROAD I wonder if a liveness / readiness probe is causing this?

**Reply by rvowles** [reply]
Hey @reubenhEROAD - you used the standard helm chart?

**Comment by rvowles** [comment]
If so, please go into  the global.extra.extraCommonConfigFiles and there is a line in the logs that is commented out:

<AsyncLogger name="jersey-logging" level="trace"/>

move that line down under the --> and then redeploy

That will cause your jersey traffic to get considerably more verbose.

**Reply by reubenhEROAD** [reply, marked as answer]
verbose logging enabled now thanks @rvowles, and nice one @chrusty, your suspicions were well placed:
`responded: GET - http://10.130.21.108:32763/health-check`
alb.ingress.kubernetes.io/healthcheck-path was configured wrong, should have been /health/liveness
Also I had alb.ingress.kubernetes.io/success-codes configured to 200,404 for some daft reason, which is why the failing health didn't appear as an issue.
All fixed now.
